website/content/v1.6/talos-guides/install/bare-metal-platforms/equinix-metal.md

```diff
@@ -95,13 +97,13 @@ metal device create \
   --project-id $PROJECT_ID \
   --facility $FACILITY \
   --operating-system "custom_ipxe" \
-  --ipxe-script-url "https://pxe.factory.talos.dev/pxe/376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba/{{< release >}}/equinixMetal-amd64" \
+  --ipxe-script-url "https://pxe.factory.talos.dev/pxe/a6ef1cf923b0b123f88968fb611f4b4d5e53dd8f77be11ba010a38c4bab7f505/{{< release >}}/metal-amd64" \
   --plan $PLAN\
   --hostname $HOSTNAME\
   --userdata-file controlplane.yaml
 ```
 
-e.g. `metal device create -p <projectID> -f da11 -O custom_ipxe -P c3.small.x86 -H steve.test.11 --userdata-file ./controlplane.yaml --ipxe-script-url "https://pxe.factory.talos.dev/pxe/376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba/{{< release >}}/equinixMetal-amd64"`
+e.g. `metal device create -p <projectID> -f da11 -O custom_ipxe -P c3.small.x86 -H steve.test.11 --userdata-file ./controlplane.yaml --ipxe-script-url "https://pxe.factory.talos.dev/pxe/a6ef1cf923b0b123f88968fb611f4b4d5e53dd8f77be11ba010a38c4bab7f505/{{< release >}}/metal-amd64"`
```

Comment:
This looks wrong to me, it should be Equinix?

Reply:
we don;'t have an equinix profile in imager

Reply:
hm... I'm confused then. need to check this, we definitely need this to be a proper platform

Reply:
https://pxe.factory.talos.dev/pxe/376567988ad370138ad8b2698212367b8edcb69b5fd68c80be1f2ec7d603b4ba/v1.6.7/equinixMetal-amd64 hmm this seems to work, but it's not defined here: https://github.com/siderolabs/talos/blob/main/pkg/imager/profile/default.go :thin

Reply:
it doesn't have to be a profile - it's a platform